### PR TITLE
[admission-controller] Add https proxy management

### PR DESCRIPTION
**Pull request by miketnt**
## Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)
- [x] PR only contains changes for one chart

### CODE REVIEW

File: charts/admission-controller/values.yaml [hunk 1]
```diff
@@ -101,6 +103,8 @@ scanner:
   # Put your <Sysdig-Secure-Secret-Token> in this value
   # Set to http://proxyIp:proxyPort in case connection to Sysdig Secure requires a proxy
   httpProxy: ""
+  # Set to https://proxyIp:proxyPort in case connection to Sysdig Secure requires a proxy
```
Comment (airadier) [inline charts/admission-controller/values.yaml]: Better use plain http:// in the example...

**Review by airadier — CHANGES_REQUESTED** [review]
You'll need to update the version in Chart.yaml to pass the linter check.

File: charts/admission-controller/values.yaml (hunk 1 above)
Comment (miketnt) [reply charts/admission-controller/values.yaml]: Updated both the example and the version in the Chart. We're on the road again. Thanks @airadier!

**Review by airadier — APPROVED** [review]

**Comment by airadier** [comment]
LGTM

**Pull request merged**